views/partials/card.handlebars

```diff
@@ -8,7 +8,7 @@
         {{promise.user.username}}
       </a>
       <span class="{{scoreColor promise.user.score}}">
-        {{prettyPercent promise.user.score 2}}
+        {{prettyPercent promise.user.score 2}} ({{promise.user.doneOrDueCount}} commitments)
```

Comment:
i can play around with the UI side a bit to see if we can make things fit a little better:
![screen shot 2018-08-05 at 5 34 05 pm](https://user-images.githubusercontent.com/112371/43690284-d8335654-98d5-11e8-9f9d-ea4a14a1563e.png)

Reply:
OK, obviously needed - it wasn't quite that blatantly ugly on my test version with fewer promises :-)